gui/default/syncthing/core/editShareTemplate.html

```diff
@@ -0,0 +1,33 @@
+<div class="col-md-6 checkbox">
+  <label for="sharedwith-{{id}}" ng-switch="currentSharing.editing">
+    <input id="sharedwith-{{id}}" ng-model="currentSharing.selected[id]" type="checkbox" />
+    <span tooltip data-original-title="{{id}}">
+      <span ng-switch-when="device">{{folderLabel(id)}}</span>
+      <span ng-switch-when="folder">{{deviceName(findDevice(id))}}</span>
+    </span>
+  </label>
+</div>
+<div class="col-md-6">
+  <div class="input-group">
+    <span class="input-group-addon" ng-switch="!currentSharing.encPWs[id] || currentSharing.encPWs[id] === ''">
+      <span ng-switch-when='true' class="fas fa-fw fa-unlock" />
+      <span ng-switch-when='false' class="fas fa-fw fa-lock" />
```

Comment:
Should we show a lock when the type is receive encrypted, even though there is no password entered and one cannot be entered?]

Reply:
Probably makes sense, consistent with a lock meaning "data exchanged with this device is encrypted".